Source/FakeItEasy.sln

```diff
@@ -64,6 +64,9 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Integration", "Integration", "{5D41D463-D53B-4ADC-90AD-44AA03D6340D}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Specs", "Specs", "{575A3C3B-17BA-4A8A-9F53-05320913D0A6}"
+	ProjectSection(SolutionItems) = preProject
+		CreationOptionsSpecs.cs = CreationOptionsSpecs.cs
+	EndProjectSection

```

Comment:
Whoa. Yes. I do not know what was going on there.
Worse, I can't explain why I didn't catch it in pre-PR review. :blush: